src/tests/test_main.py

```diff
@@ -187,6 +189,18 @@ def test_expand_uris(self):
         ]
         self.assertEqual(main.expand_uris(msid, given), expected)
 
+    def test_expand_inline_graphics(self):
```

Comment:
Only inline graphics from the CDN (as we can't do anything with them, and they're tiny).